package.json

```diff
@@ -140,6 +141,7 @@
   },
   "scripts": {
     "test": "rm -rf built && jest",
+    "coverage": "yarn test --collectCoverage && codecov",
```

Comment:
Minor: what do you think of having just the `codecov` call here? Then on CI we can change the tests call to add the `--collectCoverage` flag. Just to keep those scripts atomic.

Reply:
I actually like that better but it's traditional for the coverage command to actually run the tests along generating the report, @ferhatelmas what do you think?

Reply:
IMHO it seems like a good idea at first but we run coverage only on CI so fine as it is.